meta-iotlab/recipes-devtools/openocd/openocd-ti.bb

```diff
@@ -0,0 +1,76 @@
+DESCRIPTION = "TI's fork of OpenOCD Free and Open On-Chip Debugging, In-System Programming and Boundary-Scan Testing"
+HOMEPAGE = "git://git.ti.com/sdo-emu/openocd.git"
+SECTION = "utils"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+
+PR = "r1"
+
+DEPENDS = ""
+DEPENDS += "libftdi"
```

Comment:
Maybe just use the bare minimum dependencies to be able to program TI boards ? I guess libftdi is not required ?

Reply:
true, I'll check what I can remove. I'd have to modify  the subsequent configure steps to match